app/lib/index.js

```diff
@@ -99,6 +100,10 @@ rpc.on('update available', ({ releaseName, releaseNotes }) => {
   store_.dispatch(updaterActions.updateAvailable(releaseName, releaseNotes));
 });
 
+rpc.on('move', () => {
+  store_.dispatch(uiActions.setFontSmoothing());
```

Comment:
To make this absolutely perfect, what I would do here is dispatch the *actual* action / intent that's ocurring (in this case, `UI_WINDOW_MOVE` action or similar, which as an `effect` dispatches that other action (so that it's overridable by plugins ;))